source/node/Entry.js

```diff
@@ -212,6 +212,31 @@ class Entry {
         return raw.hasOwnProperty(property) ? raw[property] : undefined;
     }
 
+    /**
+     * Get property values via RegExp expressions.
+     * If no property expression is specified, it returns the empty behavior of
+     * {@see Entry.getProperty}.
+     * @param {RegExp|String} propertyExpression
+     * @returns {Array} The values of the properties that matched the expression.
+     * @memberof Entry
+     */
+    getProperties(propertyExpression) {
+        if (typeof propertyExpression === "undefined") {
+            return this.getProperty();
+        }
+
+        const raw = this._getRemoteObject().properties;
+        var result = [];
```

Comment:
Im a bit late but shouldnt this be removed?

Reply:
It should, I opened a new PR to remove it (#241).